content/api/commands/intercept.md

```diff
@@ -170,116 +94,153 @@ The `routeHandler` defines what will happen with a request if the [routeMatcher]
+// ...but not this: https://staging.cypress.io/users
+// ...or this: http://localhost/users
+```
+Glob pattern matching provides the necessary flexibility:
+
+```js
+cy.intercept('**/users')
+// matches all of these:
+// https://prod.cypress.io/users
+// https://staging.cypress.io/users
+// http://localhost/users
 
-```javascript
-// will match any HTTP method to urls that end with 3 or 5
 cy.intercept('**/users?_limit=+(3|5)')
+// matches all of these:
+// https://prod.cypress.io/users?_limit=3
+// http://localhost/users?_limit=5
+
 ```
+Under the hood, Cypress uses the [minimatch]() library for glob matching and provides access to it via the `Cypress` global.
+This enables you to test your pattern in the Test Runner browser console.
 
-**Tip:** you can evaluate your URL using DevTools console to see if the [`minimatch` pattern](https://www.npmjs.com/package/minimatch) is correct.
+The first argument that the `Cypress.minimatch` function takes is the URL (string) and the second is the pattern (string). If invoking the function with those yields `true`, then you have a match!
 
 ```javascript
-// paste into the DevTools console while Cypress is running
-Cypress.minimatch(
-  'https://jsonplaceholder.cypress.io/users?_limit=3',
-  '**/users?_limit=+(3|5)'
-) // true
-
-// print verbose debug information
-Cypress.minimatch(
-  'https://jsonplaceholder.cypress.io/users?_limit=3',
-  '**/users?_limit=+(3|5)',
-  { debug: true }
-) // true + lots of debug messages
+// executed in the Test Runner browser console:
+Cypress.minimatch('http://localhost/users?_limit=3', '**/users?_limit=+(3|5)') 
+// true
+Cypress.minimatch('http://localhost/users?_limit=5', '**/users?_limit=+(3|5)') 
+// true
+Cypress.minimatch('http://localhost/users?_limit=7', '**/users?_limit=+(3|5)') 
+// false
 ```
 
-You can even add an assertion to the test itself to ensure the URL is matched
+The third argument that `Cypress.minimatch` takes is for specifying options (object), one of which is `debug` which if set to `true`, will yield verbose output that could help you understand why your pattern isn't working as you expect:
 
-```javascript
-// arguments are url and the pattern
-expect(
-  Cypress.minimatch(
-    'https://jsonplaceholder.cypress.io/users?_limit=3',
-    '**/users?_limit=+(3|5)'
-  ),
-  'Minimatch test'
-).to.be.true
+```js
+Cypress.minimatch('http://localhost/users?_limit=3', '**/users?_limit=+(3|5)', { debug: true }) 
+// true (plus debug messages)
 ```
 
-For the most powerful matching, provide a regular expression
+### Specifying a Method
 
-```javascript
-cy.intercept(/\/users\?_limit=(3|5)$/).as('users')
-cy.get('#load-users').click()
-cy.wait('@users').its('response.body').should('have.length', 3)
+If you don't pass a `method` argument, then all HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, `DELETE`, etc.) will match.
 
-// intercepts _limit=5 requests
-cy.get('#load-five-users').click()
-cy.wait('@users').its('response.body').should('have.length', 5)
-```
```

Comment:
I decided to omit this example since I don't think matching with regex should be encouraged. There's still a line in a previous snippet showing that regex is an option.